packages/components/src/components/domainproperties/dataset/DatasetPropertiesPanel.tsx

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c) 2020 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Form, Row, Col } from 'react-bootstrap';
+import { Utils } from '@labkey/api';
+import { DomainPanelStatus } from "../models";
+import { DataRowUniquenessContainer, BasicPropertiesFields } from "./DatasetPropertiesPanelFormElements";
+import { AdvancedSettings } from "./DatasetPropertiesAdvancedSettings";
+import { DatasetModel} from "./models";
+import { InjectedDomainPropertiesPanelCollapseProps, withDomainPropertiesPanelCollapse } from "../DomainPropertiesPanelCollapse";
+import { BasePropertiesPanel, BasePropertiesPanelProps } from "../BasePropertiesPanel";
+
+interface OwnProps {
+    model: DatasetModel;
+    panelStatus?: DomainPanelStatus;
+    onChange?: (model: DatasetModel) => void;
+    controlledCollapse?: boolean;
+    initCollapsed?: boolean;
+    collapsible?: boolean;
+    onToggle?: (collapsed: boolean, callback: () => any) => any;
+    validate?: boolean;
+    useTheme?: boolean;
+    successBsStyle?: string;
+    newDataset: boolean;
```

Comment:
See comments from the advanced settings component. I think we can determine if the dataset is new or existing from the model property and don't need a separate property for this.